animation_nodes/math/matrix.pyx

```diff
@@ -316,3 +316,10 @@ cdef float getMatrix3x3PartDeterminant(Matrix3_or_Matrix4 *m):
         m.a12 * m.a21 * m.a33 -
         m.a11 * m.a23 * m.a32
     )
+
+cdef void createMatrix(Matrix4 *m, Vector3 *center, Vector3 *tangent,
```

Comment:
Rename this to `matrixFromNormalizedAxisData`. Also in the pxd file.

Reply:
Okay.

Reply:
Done.